### PR TITLE
Verify email

### PR DESCRIPTION
**Pull request by pradel**
- generateVerificationEmailToken(userId: string, email: string) save a new token in the db (not sure about the name of the function)
- verifyEmail(token: string) verify the email

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -125,6 +126,41 @@ class Mongo {
     }
   }
 
+  async generateVerificationEmailToken(userId: string, email: string): Promise<void> {
+    const ret = await this.collection.update({ _id: userId }, {
+      $push: {
+        'services.email.verificationTokens': {
+          token: crypto.randomBytes(43).toString('hex'),
```
Comment (pradel) [inline src/index.js]: I used crypto, but maybe it's better to use a `@accounts/server` function?

File: src/index.js [hunk 2]
```diff
@@ -125,6 +126,41 @@ class Mongo {
     }
   }
 
+  async generateVerificationEmailToken(userId: string, email: string): Promise<void> {
```
Comment (pradel) [inline src/index.js]: `email` should be an unverified email or throw if the user doesn't have an unverified email. This should be checked on Accounts server or here?

File: src/index.js (hunk 1 above)
Comment (TimMikeladze) [inline src/index.js]: Yes, the token should be generated in `@accounts/server` and then passed down to the database method call.

File: src/index.js (hunk 2 above)
Comment (TimMikeladze) [inline src/index.js]: The majority of the logic should occur in the core accounts package. The database packages should handle the bare minimum (writes and reads).

**Pull request closed without merging**